Websites/bugs.webkit.org/skins/custom/global.css

```diff
@@ -2452,41 +2475,176 @@ form table td {
 .create-a-new-bugzilla-account main > section,
 .log-in-to-bugzilla main > section,
 .choose-account main > section {
-   max-width: 75ch;
-   margin: 0 auto 33vh;
+    max-width: 75ch;
+    margin: 0 auto 33vh;
 }
 
 .center-align {
-   text-align: center;
+    text-align: center;
 }
+
 .left-align {
-   text-align: center;
+    text-align: center;
 }
+
 .right-align {
-   text-align: center;
+    text-align: center;
 }
 
 /** Login page **/
 form[name=login] table,
 form[name=login] table tr {
-   border: 0;
+    border: 0;
 }
 
 .yui-skin-sam .yui-ac-content {
     background-color: var(--input-background-color);
     border-color: var(--input-border-color);
 }
 
+/** Attachment File Diff **/
```

Comment:
Ditto.

Reply:
Rebased the patch against main to ensure everything is correctly modified.